Firestore/Source/API/FIRFirestore.mm

```diff
@@ -97,6 +97,8 @@ + (void)initialize {
   if (self == [FIRFirestore class]) {
     SetThrowHandler(ObjcThrowHandler);
   }
+
+  Firestore::SetClientLanguageHeader("gl-objc/2.0");
```

Comment:
Feel free to suggest a better call site for this -- I'm not sure this is a particularly great place.

Reply:
This is the place to do it. However, it should be inside the `if` block (which is a defense against this running more than once).

Reply:
Done.